src/devices/ExplorerHat/DCMotorExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.ExplorerHat
+{
+    /// <summary>
+    /// Extensions methods to operate with DCMotors
+    /// </summary>
+    public static class DCMotorExtensions
+    {
+        /// <summary>
+        /// Motor turns forwards at indicated speed
+        /// </summary>
+        /// <param name="motor">DCMotor instance to extend</param>
+        /// <param name="speed">Indicated speed</param>
+        public static void Forwards(this DCMotor.DCMotor motor, double speed = 1)
+        {
+            motor.Speed = speed;
+        }
+
+        /// <summary>
+        /// Motor turns backwards at indicated speed
+        /// </summary>
+        /// <param name="motor">DCMotor instance to extend</param>
+        /// <param name="speed">Indicated speed</param>
+        public static void Backwards(this DCMotor.DCMotor motor, double speed = 1)
+        {
+            motor.Speed = Math.Abs(speed) * -1;
```

Comment:
I'd prefer we just used `-speed`. I.e. for Forward we will still move backwards if negative speed is passed and that makes sense to me so for Backwards I'd expect it to work same as `Forwards(-speed)`

Reply:
Oh, I like backwards feature, I think that it helps working on two wheels rovers with kids... But I understand your intend to simplifies API. I'll remove it. Anyway, it can be implemented by clients. In that case, would you change the name Forwards by Turn, or Move, or ???...

Reply:
I only meant to not do ABS, Backwards is fine